frontend/app/component/form/testHelperFunctions.tsx

```diff
@@ -0,0 +1,31 @@
+import { within } from "@testing-library/react";
+import { expect } from "vitest";
+
+export function expectFieldsToBeInvalidAndToHaveAccessibleErrorMessage(
+  fields: Array<HTMLElement>,
+  feedbackMessage: string,
+) {
+  fields.forEach((field) => {
+    expect(field).toBeInvalid();
+    expect(field).toHaveAccessibleErrorMessage(feedbackMessage);
+  });
+}
+
+export function expectFieldsToHaveIconAndToHaveAccessibleName(fields: Array<HTMLElement>, accessibleName: string) {
+  fields.forEach((field) => {
+    expect(within(field.previousElementSibling as HTMLElement).getByRole("img")).toHaveAccessibleName(accessibleName);
```

Comment:
Am not happy about the `previousElementSibling`. Am looking into alternative solutions, might need some changes to `InputGridRow` or `FormField`.

Reply:
React Testing Library and tables are not the easiest combination. Would be nice if you could select a cell based on `rowheader` and `columnheader`, but alas.

Best solution I can come up with is to put an id on the table cell (`cell-<input field id>`), similar to the `row-<input-field-id>` we have already. If we put the `id`s in some `const`s it should be maintainable.

It's a straightforward refactor but will take some effort. So that raises the question, considering the other open PRs: should we do that in this PR or in a next one? As long as the DOM structure does not change, the current solution works. So we could merge this PR and pick up the refactor next week.

Reply:
Discussed in standup. This PR can be merged. I've created a new issue https://github.com/kiesraad/abacus/issues/329 for improving the locator, to be picked up next week.